src/greeklt/greek_iso_transliteration.py

```diff
@@ -0,0 +1,272 @@
+def greek_iso_transliteration(string: str):
+#   This should all probably be a CSV or something
+    el_low = [
+        "α",
+        "β",
+        "γ",
+        "δ",
+        "ε",
+        "ζ",
+        "η",
+        "θ",
+        "ι",
+        "κ",
+        "λ",
+        "μ",
+        "ν",
+        "ξ",
+        "ο",
+        "π",
+        "ρ",
+        "σ",
+        "τ",
+        "υ",
+        "φ",
+        "χ",
+        "ψ",
+        "ω"
+    ]
+    el_cap = [
+        "Α",
+        "Β",
+        "Γ",
+        "Δ",
+        "Ε",
+        "Ζ",
+        "Η",
+        "Θ",
+        "Ι",
+        "Κ",
+        "Λ",
+        "Μ",
+        "Ν",
+        "Ξ",
+        "Ο",
+        "Π",
+        "Ρ",
+        "Σ",
+        "Τ",
+        "Υ",
+        "Φ",
+        "Χ",
+        "Ψ",
+        "Ω"
+    ]
+    # TODO FIX Psari, not PSari
+    eng_low = [
+        "a",
+        "v",
+        "g",
+        "d",
+        "e",
+        "z",
+        "i",
+        "th",
+        "i",
+        "k",
+        "l",
+        "m",
+        "n",
+        "x",
+        "o",
+        "p",
+        "r",
+        "s",
+        "t",
+        "y",
+        "f",
+        
+        "ch",
+        "ps",
+        "o"
+    ]
+    eng_cap = [
+        "A",
+        "V",
+        "G",
+        "D",
+        "E",
+        "Z",
+        "I",
+        "TH",
+        "I",
+        "K",
+        "L",
+        "M",
+        "N",
+        "X",
+        "O",
+        "P",
+        "R",
+        "S",
+        "T",
+        "Y",
+        "F",
+        "CH",
+        "PS",
+        "O"
+    ]
+    # Simple digraphs with no extra rules or edge cases
+    # No need for mixed casing "Γγ" or "Γξ" against Greek phonology
+    el_simple_digraphs = [
+        "γγ",
+        "γξ",
+        "γχ"
+    ]
+    el_simple_cap_digraphs = [
+        "ΓΓ",
+        "ΓΞ",
+        "ΓΧ"
+    ]
+    eng_simple_digraphs = [
+        "ng",
+        "nx",
+        "nch"
+    ]
+    eng_simple_cap_digraphs = [
+        "NG",
+        "NX",
+        "NCH"
+    ]
+
+    el_mono_digraph_sub = [
+        "Θ",
+        "Χ",
+        "Ψ"
+    ]
+#   Accent based digraphs
+#    el_low_acc_digraphs = [
+#        "άυ",
+#        "αϋ",
+#
+#        "έυ",
+#        "εϋ",
+#
+#        "ήυ",
+#        "ηϋ"
+#
+#    ]
+#    el_mix_acc_digraphs = [
+#        "Άυ",
+#        "Αϋ",
+#
+#        "Έυ",
+#      "Εϋ",
+#
+#        "Ήυ",
+#        "Ηϋ"
+#    ]
+#    el_cap_acc_digraphs = [
+#        "ΆΥ",
+#        "ΑΫ",
+#
+#        "ΈΥ",
+#        "ΕΫ",
+#
+#        "ΉΥ",
+#        "ΗΫ"
+#    ]
```

Comment:
As for accents, I think we could drop all accentuation like here: https://www.passport.gov.gr/passports/GrElotConverter/GrElotConverter.html

From `ψάρι` to `psari` and not `psάri`.

Reply:
For now, in the interests of getting it finished I decided to fully drop the accentuation as a feature. However the ELOT 743 standard actually does have a table for non-character conversions, and ultimately I think we should conform to the standard as close as possible. Maybe have accent conversion be an option or something.

Reply:
The standard has the accentuation but my intention was to drop the altogether because I haven't ever found a use for it. Maybe we could implement it later!

Reply:
Fair! It's a very limited use case and even theoretically I can't see an actual use for it. It'd be a fun side project though.

Safe to say we've resolved this issue as a future TBD; I'll leave the commented out lines in place to expand upon them in the future